src/mui/input/ReferenceInput.js

```diff
@@ -288,13 +263,12 @@ function mapStateToProps(state, props) {
     };
 }
 
-const ConnectedReferenceInput = connect(mapStateToProps, {
-    crudGetOne: crudGetOneAction,
-    crudGetMatching: crudGetMatchingAction,
-})(ReferenceInput);
-
-ConnectedReferenceInput.defaultProps = {
-    addField: true,
-};
+const enhance = compose(
+    connect(mapStateToProps, {
+        crudGetOne: crudGetOneAction,
+        crudGetMatching: crudGetMatchingAction,
+    }),
+    translate
+);
 
-export default ConnectedReferenceInput;
+export default enhance(ReferenceInput);
```

Comment:
The exported component needs to define its default props, with addField: true in it